### DISCUSSION TITLE
Prevent function names mangling

### DISCUSSION THREAD

**Discussion by VladimirMikulic** [parcel-bundler/parcel#4972] [General]
Hi Parcel team,

I am using parcel v1.12.4 and I would like to prevent function names from being mangled.
Does Parcel support this?

Thank you,

**Comment by steve-taylor** [comment]
Rollup produces much more readable output if that’s what you’re looking for. It also does scope hoisting, just like Parcel. Unlike Parcel, it’s not trivial to configure, but there are some nice wrappers such as Vite, Microbundle and Preconstruct.
